apps/teamsjs-diagnostic-app/src/components/sample/lib/config.ts

```diff
@@ -0,0 +1,8 @@
+const config = {
```

Comment:
Why do we need to set env variables is there another way to do this?

Reply:
I see! it seems to be an unnecessary file for the purpose of my app so i got rid of it!